src/app/dynamic/enterprise/metering/template.html

```diff
@@ -26,6 +26,18 @@
 </ng-container>
 
 <ng-container *ngIf="isLoading()">
```

Comment:
```suggestion
<ng-container *ngIf="isLoading() || !loadOldReportsCard">
```

And then we can remove the spinner below. Wdyt?

Reply:
Makes sense, I like this idea 👍

Reply:
I had to tweak it a bit cos there is a possibility that there are no old reports in which case we do not load the component.